frontend/src/app/shared/pipes/cater-capacity.pipe.ts

```diff
@@ -0,0 +1,30 @@
+import type { PipeTransform } from '@angular/core';
+import { Pipe } from '@angular/core';
+import type { CanCaterFor } from 'digital-fuesim-manv-shared';
+
+export type CatererType = 'material' | 'personnel';
+
+@Pipe({
+    name: 'caterCapacity',
+})
+export class CaterCapacityPipe implements PipeTransform {
+    transform(canCaterFor: CanCaterFor, type: CatererType): string {
```

Comment:
There should be some kind of shortcut with a different text when the provided `canCaterFor` cannot treat anyone. You can see this with the `gf`. It is a bit weird to read that they can help patients of different injury levels, but in fact, they can't help anyone.